dataclass_mapper/mapping_method.py

```diff
@@ -120,10 +120,10 @@ def __init__(self, source_cls: ClassMeta, target_cls: ClassMeta) -> None:
         self.source_cls = source_cls
         self.target_cls = target_cls
         self.lines = [
-            f'def convert(self, extra: dict) -> "{self.target_cls.name}":',
+            f'def convert(self, extra: Dict) -> "{self.target_cls.name}":',
```

Comment:
That should fix the problem.

```suggestion
            'from typing import Dict',
            f'def convert(self, extra: Dict) -> "{self.target_cls.name}":',
```

Reply:
Or alternatively, just remove the typing annotation here. It's not used anyhow.

In either case, this will also break some tests in `test_mapping_method`.

Reply:
I replaced `Dict` with `dict`, and that fixed the aforementioned issue!